src/core/operations/ParseCSR.mjs

```diff
@@ -233,23 +268,79 @@ function describeKeyUsage(extension) {
 function describeExtendedKeyUsage(extension) {
     const usage = [];
 
-    if (extension.serverAuth) usage.push("TLS Web Server Authentication");
-    if (extension.clientAuth) usage.push("TLS Web Client Authentication");
-    if (extension.codeSigning) usage.push("Code signing");
-    if (extension.emailProtection) usage.push("E-mail Protection (S/MIME)");
-    if (extension.timeStamping) usage.push("Trusted Timestamping");
-    if (extension.msCodeInd) usage.push("Microsoft Individual Code Signing");
-    if (extension.msCodeCom) usage.push("Microsoft Commercial Code Signing");
-    if (extension.msCTLSign) usage.push("Microsoft Trust List Signing");
-    if (extension.msSGC) usage.push("Microsoft Server Gated Crypto");
-    if (extension.msEFS) usage.push("Microsoft Encrypted File System");
-    if (extension.nsSGC) usage.push("Netscape Server Gated Crypto");
+    const ekuIdentifierToName = new Map([
```

Comment:
What's the advantage of using a map here, over a simple object?

Reply:
Nothing really, refactored the code to use Object instead.